data-prepper-expression/src/main/java/org/opensearch/dataprepper/expression/LowerThanOperator.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.expression;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+public class LowerThanOperator implements Operator<Boolean> {
```

Comment:
Nit: I think `LessThan` or `LessThanOrEqual` are a little better names for these